invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -168,8 +193,8 @@ export class DeleteCommunityModal extends Component {
                     /* eslint-disable-next-line jsx-a11y/label-has-associated-control */
                     <label>
                       <Trans>
-                        <strong>All the members</strong> will be removed from the
-                        community.
+                        <strong>{`${membersCount.toString()} members`}</strong> will be
```

Comment:
Changed it to 

```javascript
<strong>{`${membersCount}`} members< /strong> will be
```

Otherwise, the `membersCount` value wouldn't print even if it was in the state. Another alternative is the `toString()` method.